src/cms/api/redirects/models/redirects.settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "kind": "collectionType",
+  "collectionName": "redirects",
+  "info": {
+    "name": "Redirects"
```

Comment:
If you change the name to legacy-redirect above, then this should be changed to LegacyRedirects